### ISSUE TITLE
Simple Transition Animation

### ISSUE THREAD

**Issue by carbureted** [newbreedofgeek/react-stepzilla#49]
A fade-in effect between screens would be awesome.

**Comment by newbreedofgeek** [comment]
@carbureted good idea. You may be able to do this usinghttps://github.com/reactjs/react-transition-group. Would be great if you can see if it's possible and let us know, or raise a PR.
